pkg/alertmanager/state_persister_test.go

```diff
@@ -82,12 +83,13 @@ func makeTestFullState() *clusterpb.FullState {
 }
 
 func makeTestStatePersister(t *testing.T, position int, userID string) (*fakePersistableState, *fakeStore, *statePersister) {
+	reg := prometheus.NewPedanticRegistry()
```

Comment:
Why make this registry, if it's not tested? (We can pass `nil` as registry to `newStatePersister`)